index.js

```diff
@@ -10,3 +21,124 @@ document.querySelectorAll('.nav-menue > a').forEach((n) => n.addEventListener('c
   humburgerBtn.classList.remove('active');
   navMenu.classList.remove('active');
 }));
+
+const arrWork = [
+  {
+    id: 1,
+    title: 'Tonic',
+    company: 'canopy',
+    role: 'Back End Dev',
+    year: 2015,
+    closeImg: 'images/close.jpg',
+    img: 'images/web-img1.png',
```

Comment:
- [ ] You did great storing the information in an array of objects! 💯 However, you missed an important part of the requirements:

![image](https://user-images.githubusercontent.com/77069011/202475399-9d541a6a-3867-4932-b1cf-6cb33813bba2.png)

The projects section should be dynamically displayed when the page loads, this will ensure that you have a single source of data for both your project section and the popup window, and you wouldn't need to write static text in your HTML, providing an easy way of adding more projects in the future! 🤓 🛠️ 

💡 _Hint: You could use this array of objects with the HTML that you have for the project section, iterate through it, and build a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) with the information of each project. Then you could append this template string to the `<section>` tag which contains the projects and achieve this dynamic display that is required._